portal-bridge/src/pandaops.rs

```diff
@@ -45,6 +47,23 @@ impl PandaOpsMiddleware {
     /// multiple async requests. Using "ureq" consistently resulted in errors as soon as the number of
     /// concurrent tasks increased significantly.
     pub async fn batch_request(&self, obj: Vec<JsonRequest>) -> anyhow::Result<String> {
+        let futs = obj
+            .chunks(BATCH_LIMIT)
+            .map(|chunk| self.batch_request_100(chunk.to_vec()))
+            .collect::<Vec<_>>();
+        match futures::future::join_all(futs).await.into_iter().try_fold(
+            Vec::new(),
+            |mut acc, next| {
+                acc.extend_from_slice(&next?);
+                Ok::<Vec<Value>, Box<dyn std::error::Error>>(acc)
+            },
+        ) {
+            Ok(val) => Ok(serde_json::to_string(&val)?),
+            Err(err) => Err(anyhow!("Unable to flatten batch request: {err:?}")),
+        }
+    }
+
+    async fn batch_request_100(&self, obj: Vec<JsonRequest>) -> anyhow::Result<Vec<Value>> {
```

Comment:
if we change the `BATCH_LIMIT` const this function name will not be relevant anymore.